runtime/src/inline_spl_associated_token_account.rs

```diff
@@ -0,0 +1,6 @@
+// Partial SPL Associated Token Account declarations inlined to avoid an external dependency on the spl-associated-token-account crate
+solana_sdk::declare_id!("ATokenGPvbdGVxr1b2hvZbsiqW5xWH25efTNsLJA8knL");
+
+pub(crate) mod new_program {
```

Comment:
tiny nit: I think it's helpful to clearly identity the program version here:
```suggestion
pub(crate) mod program_v1_0_4 {
```